README.md

```diff
@@ -28,59 +28,50 @@ GameDig.query({
-* **ipFamily**: number - IP family/version returned when looking up hostnames via DNS, can be 0 (IPv4 and IPv6), 4 (IPv4 only) or 6 (IPv6 only). (default 0)
-* **debug**: boolean - Enables massive amounts of debug logging to stdout. (default false)
-* **requestRules**: boolean - Valve games only. Additional 'rules' may be fetched into the `raw` field. (default false)
-* **requestRulesRequired**: boolean - Valve games only. `requestRules` is always required to have a response or the query will timeout. (default false)
-* **requestPlayersRequired**: boolean - Valve games only. Querying players is always required to have a response or the query will timeout. Some [games](GAMES_LIST.md) may not provide a players response. (default false)
-
-## Return Value
+## Query Response
 
 The returned state object will contain the following keys:
 
-* **name**: string - Server name
-* **map**: string - Current server game map
-* **password**: boolean - If a password is required
-* **numplayers**: number
-* **maxplayers**: number
-* **players**: array of objects - note that this could be of a different length compared to **numplayers**
-  * **name**: string - If the player's name is unknown, the string will be empty.
-  * **raw**: object - Additional information about the player if available.
-* **bots**: array of objects - Same schema as `players`
-* **connect**: string
-  * This will typically include the game's `ip:port`
-  * The port will reflect the server's game port, even if your request specified the game's query port in the request.
-  * For some games, this may be a server ID or connection url if an IP:Port is not appropriate for end-users.
-* **ping**: number
-  * Round trip time to the server (in milliseconds).
-  * Note that this is not the RTT of an ICMP echo, as ICMP packets are often blocked by NATs and node
-    has poor support for raw sockets.
-  * This value is derived from the RTT of one of the query packets, which is usually quite accurate, but may add a bit due to server lag.
-* **raw**: object
-  * Contains all information received from the server in a disorganized format.
+| Key |   | Type | Description |
+|:---|:---|:---|:---|
+| **name** |   | string | Server name. |
+| **map** |   | string | Server map. |
+| **password** |   | boolean | If a password is required. |
+| **numplayers** |   | number | Number of players connected, via [A2S_INFO](https://developer.valvesoftware.com/wiki/Server_queries#A2S_INFO). |
+| **maxplayers** |   | number | Maximum number of connected players. |
+| **players** |   | array of objects | Note that this could be of a different length compared to **numplayers**. Via [A2S_PLAYER](https://developer.valvesoftware.com/wiki/Server_queries#A2S_PLAYER). |
+| **players** | **name** | string | If the player's name is unknown, the string will be empty. |
+| **players** | **raw** | object | Additional information about the player if available. |
```

Comment:
I think this could be simpler to just do `players.name` rather than making another column for child properties.

Reply:
I agree! It was like that before, but I decided to complicate a bit... 😅